### PR TITLE
FLW-60 Immunization module enhancement UI and functionality both are done.

### PR DESCRIPTION
**Pull request by ritik-bizbrolly**
Categories the Beneficiary data according to Birth Age and add some indicators on Vaccination list as per the due , overdue and missed.

### CODE REVIEW

File: app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt [hunk 1]
```diff
@@ -312,6 +314,103 @@ fun filterBenHRPTFormList(
...
+        var seventhVal = ""
+        var eighthVal = ""
+
+        if (filterText.contains("5-6 years")) {
+            firstVal =   "${filterText.split("-")[0]} years"
+            filterText = filterText.split("-")[1]
+
+        } else if (filterText.contains("16-24 months")) {
+            firstVal = "1 years"
+            /*secondVal = "17 months"
```
Comment (drtechie) [inline app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt:338]: Remove commented code

**Comment by drtechie** [comment]
Remove commented code.
Remove unnecessary line breaks.

File: app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt [hunk 2]
```diff
@@ -312,6 +314,103 @@ fun filterBenHRPTFormList(
...
+
+        }
+        else if (filterText.contains("14 weeks")) {
+
+            filterText = "4 months"
+
+        }
+        return list.filter {
+
+            filterForImm(
```
Comment (drtechie) [inline app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt:372]: What are these "val" variables? Doesn't seem to be intuitive.

Comment (ritik-bizbrolly) [reply app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt:347]: Sorry and  I have changed this and also removed all the unnecessary comment and line breaks.

File: app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt (hunk 1 above)
Comment (ritik-bizbrolly) [reply app/src/main/java/org/piramalswasthya/sakhi/helpers/CommonUtils.kt]: Removed.

**Comment by drtechie** [comment]
Not sure why this PR was merged without approval. 
My question was regarding the argument naming.

**Pull request merged**